### PR TITLE
Fix a couple of problems detected by codespell

### PR DESCRIPTION
**Pull request by ericdlaspe**
There are a few "problems" detected by codespell that are false-positives (should be ignored), but these two are true-positives.

### CODE REVIEW

**Comment by seancorfield** [comment]
I don't think there should be a `,` after `Below` in that change. It would be better to reword it to:
```
Some common, idiomatic aliases are shown below.
```

**Comment by ericdlaspe** [comment]
Reading it it again, I agree. I'll change it to your suggestion.
